extensions/laravel-forge/src/Site.tsx

```diff
@@ -116,6 +128,18 @@ export const SitesSingleView = ({ site, server }: { site: ISite; server: IServer
               </ActionPanel>
             }
           />
+          <List.Item
+            id="edit-site-env"
+            key="edit-site-env"
+            title="Edit .env file"
+            icon={Icon.Globe}
+            accessoryTitle="press to edit in browser"
+            actions={
+              <ActionPanel>
+                <OpenInBrowserAction url={`https://forge.laravel.com/servers/${server.id}/sites/${site.id}/environment`} />
+              </ActionPanel>
+            }
+          />
```

Comment:
It seems like this would fit better as a sub action to the view .env file. What do you think?

Reply:
Sure, that makes sense